gcc/rust/resolve/rust-late-name-resolver-2.0.cc

```diff
@@ -250,6 +250,11 @@ visit_identifier_as_pattern (NameResolutionContext &ctx,
 void
 Late::visit (AST::IdentifierPattern &identifier)
 {
+  if (identifier.has_subpattern ())
```

Comment:
Do we really need a condition check over this ? Calling `DefaultResolver::visit(identifier)` directly would help future changes and let the condition in the default visitor.